modules/adding-an-odh-instance.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='adding-an-odh-instance_{context}']
+= Adding an Open Data Hub instance 
+
+[role='_abstract']
+Add an Open Data Hub instance to your project to be able to give access to an Open Data Hub dashboard.
+
+.Prerequisites
+* You have installed the Open Data Hub Operator.
+* You have created a new project for your instance of Open Data Hub.
+
+.Procedure
+. In the OpenShift web console, select *Operators* -> *Installed Operators*.
+. Click *Project* and select the *odh* project.
+. Find and click the *Open Data Hub Operator* to display the details for the currently installed version.
+. In the *KfDef* tile, click *Create instance*. `KfDef` is a specification designed to control the provisioning and management of Kubeflow deployment. The default `KfDef` configuration deploys the latest release of the Open Data Hub core components. For more information, see link:https://opendatahub.io/docs/tiered-components[Tiered Components].
+. On the *Create KfDef* page, leave the default name of *opendatahub* and click *Create*. This creates an Open Data Hub kfdef object named *opendatahub* and begins the rollout of the components.
+
```

Comment:
On the *Create KfDef* page, leave *opendatahub* as the name.  Click *Create* to create an Open Data Hub kfdef object named *opendatahub* and begin the rollout of the components.